docs/src/lib/utilities.md

```diff
@@ -9,14 +8,13 @@ CurrentModule = DataFrames
 Pages = ["utilities.md"]
 ```
 
-...
-
 ```@docs
 eltypes
 head
 completecases
-completecases!
```

Comment:
Shouldn't we be removing `completecases` here in addition to `completecases!`?

Reply:
nvm looks like they do different things; the former identifies complete cases as a boolean vector and the latter is now `dropnull!`.